### ISSUE TITLE
Random crash when deleting a mail

### ISSUE THREAD

**Issue by thgoebel** [ProtonMail/proton-mail-android#24]
Occasionally, when I delete an email in the main screen, ProtonMail crashes. Unfortunately, I fail to reproduce it consistently, but it happened to me three times today (and also previously, but now I am bothered enough to open an issue). 
Also, I'm not even sure whether this is a ProtonMail or Room issue, you tell me.

**Steps I did when it happens:**

1. Opened the app when it had been in the background for a long time
2. Open and read a new email
3. Go back to the main screen
4. Long press the just read email (i.e. the topmost in the list)
5. Click on the trash icon in the toolbar to delete the mail
6. App force closes

Android 9.0 / LineageOS 16.0 (no GApps)
ProtonMail v1.13.1 (711)

**Logcat:**

```
FATAL EXCEPTION: arch_disk_io_1
Process: ch.protonmail.android, PID: 16974
java.lang.RuntimeException: Exception while computing database live data.
	at androidx.room.o$a.run(RoomTrackingLiveData.java:6)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.IllegalStateException: Couldn't read row 75, col 1 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
	at android.database.CursorWindow.nativeGetString(Native Method)
	at android.database.CursorWindow.getString(CursorWindow.java:465)
	at android.database.AbstractWindowedCursor.getString(AbstractWindowedCursor.java:51)
	at ch.protonmail.android.api.models.room.messages.MessagesDatabase_Impl$18.call(MessagesDatabase_Impl.java:39)
	at ch.protonmail.android.api.models.room.messages.MessagesDatabase_Impl$18.call(MessagesDatabase_Impl.java:1)
	at androidx.room.o$a.run(RoomTrackingLiveData.java:5)
	... 3 more
```

**Comment by 4face-studi0** [comment]
Thank you for the report, the team was dealing with this bug, not sure whether this particular scenario has been covered. 
@dimitar1993

**Comment by ZoricaStojchevska** [comment]
Since you are experiencing this issue you can also contact our customer support at contact@protonmail.com. 
But yes the team is already addressing this issue, and this scenario is also included.

**Comment by thgoebel** [comment]
Not sure how customer support would help me, I can just reopen the app :)
Knowing that it's tracked and will eventually be fixed is fine by me.

**Comment by mpemer** [comment]
Aside from this ticket, is there some other place one might monitor this defect? Release notes, some bug tracking system like jira, or similar? (I am experiencing exactly the same as reported, but consistently every time I hit the delete button on an email. If I want to delete email without ux issues, I am currently relegated to the web app). 

Thank you!

**Comment by jwflory** [comment]
I can also confirm this bug on two other Android 9 devices with ProtonMail 1.13.1. On my device, _moto x4_, this happens periodically. It also happens sometimes when I move emails to other folders. I also have a less technical friend, Alice, who experiences this on a _Pixel 3 XL_ (Android 9, ProtonMail 1.13.1).

**Comment by jwflory** [comment]
Oops, I forgot I already filed this bug in #14. This is technically a duplicate.

**Comment by mpemer** [comment]
This happens to me too, primarily when deleting email. It always happens when I delete using the trash can icon in message view. It occasionally happens when I swipe to delete in folder view. The crash does not occur instantaneously, but after delete action is done and at some point during refresh.

**Comment by 4face-studi0** [comment]
Did you guys check that on latest Beta releases?

**Comment by jwflory** [comment]
I am running App Version 1.13.8 (726), but I have not had a crash after deleting an email for a while. Not sure about others.

**Comment by ZoricaStojchevska** [comment]
This was addressed in version 1.13.6 (723), so on the current production version you shouldn't experience this problem.

**Comment by jwflory** [comment]
Cool! For me, I might guess this issue could be closed as fixed.
